stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -4240,22 +4203,19 @@ internal enum _Variant${Self}Storage<${TypeParametersDecl}> : _HashStorage {
 %elif Self == 'Dictionary':
     let oldValue = nativeStorage.value(at: index.offset)
 %end
-    nativeDeleteImpl(nativeStorage, idealBucket: idealBucket,
+    nativeDelete(nativeStorage, idealBucket: idealBucket,
       offset: index.offset)
     return oldValue
   }
 
   internal mutating func nativeRemove(
     at nativeIndex: NativeIndex
   ) -> SequenceElement {
-    var nativeStorage = asNative
 
     // The provided index should be valid, so we will always mutating the
     // set storage.  Request unique storage.
-    let (reallocated, _) = ensureUniqueNativeStorage(nativeStorage.capacity)
-    if reallocated {
-      nativeStorage = asNative
```

Comment:
I believe this is no longer relevant with the double-indirection model removed. It was probably fixing up some redundant encoding? That or it was gibberish.